### PR TITLE
add config argument bounding box loss

### PR DESCRIPTION
**Pull request by kukientinhky**

### CODE REVIEW

File: yoloxyz/train.py [hunk 1]
```diff
@@ -268,7 +268,7 @@ def train(hyp, opt, device, tb_writer=None):
     model.gr = 1.0  # iou loss ratio (obj_loss = 1.0 or iou)
     model.class_weights = labels_to_class_weights(dataset.labels, nc).to(device) * nc  # attach class weights
     model.names = names
-
+    model.iou_loss = opt.loss_bbox
```
Comment (tienkng) [inline yoloxyz/train.py]: Tại sao argument là loss-bbox giá trị defaul là EIoU nhưng chỗ này là iou_loss vậy em

File: yoloxyz/multitasks/utils/loss.py [hunk 2]
```diff
@@ -140,7 +140,7 @@ def __init__(self, model, autobalance=False, kpt_label=False, detect_layer=None)
                 break
         self.balance = {3: [4.0, 1.0, 0.4]}.get(det.nl, [4.0, 1.0, 0.25, 0.06, .02])  # P3-P7
         self.ssi = list(det.stride).index(16) if autobalance else 0  # stride 16 index
-        self.BCEcls, self.BCEobj, self.gr, self.hyp, self.autobalance = BCEcls, BCEobj, model.gr, h, autobalance
+        self.BCEcls, self.BCEobj, self.gr, self.hyp, self.autobalance, self.iou_loss = BCEcls, BCEobj, model.gr, h, autobalance, model.iou_loss
```
Comment (tienkng) [inline yoloxyz/multitasks/utils/loss.py]: chỗ này em đã test việc bật/tắt iou thì có lỗi không?

Comment (kukientinhky) [reply yoloxyz/multitasks/utils/loss.py]: dạ em có test cả 2 hàm loss rồi ạ, rất trơn tru.

File: yoloxyz/train.py (hunk 1 above)
Comment (kukientinhky) [reply yoloxyz/train.py]: dạ tại code default sài biến iou cho các hàm loss nên em để iou_loss
